source/assets/scripts/api.js

```diff
@@ -44,11 +44,12 @@ export async function search_recipe(name, intolerances) {
 // Parameter:
 // recipe_name: the recipe name 
 // it will automaticlt get and parse the recipe in local storage if it can't find it it will just print a sentence to console,.
-function get_info_localstore(recipe_name){
```

Comment:
I don’t think we should use the name of the recipe as key since there might be collision. Instead we have the hash for this purpose. But for now this isn’t too big of a deal since we will be revamping the search functionality.